pkgs/applications/graphics/akira/unstable.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, appstream-glib
+, desktop-file-utils
+, meson
+, ninja
+, pantheon
+, pkgconfig
+, python3
+, vala_0_40
+, vala-lint
+, wrapGAppsHook
+, cairo
+, glib
+, goocanvas2
+, gtk3
+, gtksourceview3
+, json-glib
+, libarchive
+, libgee
+, libxml2 }:
+
+stdenv.mkDerivation rec {
+  pname = "akira-unstable";
+  version = "2019-10-12";
+
+  repoName = "Akira";
```

Comment:
oh sorry, I actually didn't mean to suggest this. It just happened to be in that particularly expression.

Reply:
Sure, no worries. Was going to add the patch anyway :)